src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs

```diff
@@ -250,6 +255,9 @@ private void ValidateProcessedBlock(Block suggestedBlock, ProcessingOptions opti
         block.Header.ReceiptsRoot = _receiptsRootCalculator.GetReceiptsRoot(receipts, spec, block.ReceiptsRoot);
         ApplyMinerRewards(block, blockTracer, spec);
         _withdrawalProcessor.ProcessWithdrawals(block, spec);
+
+        ProcessValidatorExits(block, spec);
```

Comment:
I think it will be better (if not specified by spec) if exists will happen after processing deposits. It shouldn't matter, but better safe than sorry